src/ctapipe/containers.py

```diff
@@ -301,6 +301,10 @@ class CameraHillasParametersContainer(BaseHillasParametersContainer):
     width = Field(nan * u.m, "standard spread along the minor-axis", unit=u.m)
     width_uncertainty = Field(nan * u.m, "uncertainty of width", unit=u.m)
     psi = Field(nan * u.deg, "rotation angle of ellipse", unit=u.deg)
+    psi_uncertainty = Field(nan * u.deg, "uncertainty of psi", unit=u.deg)
+    transverse_cog_uncertainty = Field(
+        nan * u.m, "rotated centroid y coordinate uncertainty", unit=u.m
```

Comment:
I have a hard time understanding how "rotated centroid y coordinate uncertainty" and "uncertainty on the center of gravity along the transverse axis of the image" can mean the same thing, and I think the definition you give in the changelog is much clearer.

Reply:
Yeah, i would agree here, just change this to be 
```python
"uncertainty on the center of gravity along the transverse axis of the image"
```